app/src/org/commcare/activities/connect/ConnectIdNetworkHelper.java

```diff
@@ -194,6 +195,57 @@ private HashMap<String, String> getContentHeadersForXFormPost(RequestBody postBo
         return headers;
     }
 
+    private PostResult getSyncInternal(Context context, String url, AuthInfo authInfo,
+                                        Multimap<String, String> params) {
+        isBusy = true;
+        showProgressDialog(context);
+        HashMap<String, String> headers = new HashMap<>();
+        RequestBody requestBody;
+
+        //TODO: Figure out how to send GET request the right way
```

Comment:
can we define a [Retrofit Interfaces](https://square.github.io/retrofit/) for standard connect Urls in a separate class like - `ConnectRequestEndpoints` . You can define method like - `public Response<ResponseBody> getJobs(String uri, Multimap<String, String> httpParams, Map<String, String> httpHeaders) throws IOException {`  in there and define corresponding job url with annotations right in that interface.  CommCare Urls are not static (they change for every domain) which is why we don't adpot the similar strategy for CC but we don't have that constraint with connect urls.